portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,77 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var index = 0;
+const jobCount = 3;
+
+/**
+ * Set innerHTML text in work experience section
+ */
+function setContent() {
+  const jobs = ['UI Development Intern @ Accenture', 'STEP Intern @ Google', 'STEP Intern @ Google'];
```

Comment:
you might want to consider an array of objects. something that looks like

jobs = [
{name: "UI development...", dates: "Sept, 2017 - ...", summary: "Used reactjs..."},
{name: "STEP...",...},
{...},
;]

Reply:
This way you can group information together and depend on them being at the same index in each array.

you can refer to this as jobs[index].name jobs[index].dates and jobs[index].summary